ftplugin/idris2.vim

```diff
@@ -94,7 +94,8 @@ endfunction
 
 function! IdrisReload(q)
   w
-  let tc = system("idris2 --find-ipkg " . expand ('%:p') . " --client ''")
+  let file = expand ('%:p')
+  let tc = system("idris2 --find-ipkg " . file . " --client ''")
```

Comment:
I did not remove the call to `shellescape`- see https://github.com/edwinb/idris2-vim/pull/5 

Is there a difference?

Reply:
Thanks! I'll close this PR in favor of yours.